technology-overview.md

```diff
@@ -0,0 +1,84 @@
+# Technology overview
+
+[Agoric](https://agoric.com/about/) aims at enabling safe cooperation among mutually suspicious parties
+
+The way it does it is by making it easy to write secure smart contracts and making it possible to run these contracts on one or several blockchains. Agoric strategy to writing smart contracts is to write them in JavaScript using a framework called ERTP (Electronic Rights Transfer Protocol)
+
+
+## High-level view
+
+### JavaScript
+
+Agoric works to make [ECMAScript](https://www.ecma-international.org/publications/standards/Ecma-262.htm) (standard defining JavaScript) a suitable language for easily expressing secure smart contracts. Agoric works with ECMA TC39 (which includes browser makers and Node.js representatives) in that direction
+
+Agoric also works with [ECMA TC53](https://www.ecma-international.org/memento/tc53.htm) to make [SES](https://github.com/Agoric/SES) the JavaScript for embedded devices
```

Comment:
"works with" -> "participates in"

"to make SES the JavaScript for embedded devices" -> "which builds on SES as the JavaScript standard for embedded devices"